Automattic/vip
# Update the dev-env commands to follow the VIP-CLI style guide

## Description

Updated the voice, tense, and format of command descriptions, examples, and usage models for the `vip dev-env` command group to follow the VIP-CLI style guide.

## Pull request checklist

- [ n/a ] Update [SETUP.md](https://github.com/Automattic/vip-cli/blob/trunk/docs/SETUP.md#list-of-environmental-variables) with any new environmental variables.
- [ x ] Follow the [pull request checklist](https://github.com/Automattic/vip-cli/blob/trunk/docs/RELEASING.md#new-pull-requests)
- [ x ] Add/update [automated tests](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#automated-testing) as needed.

## New release checklist


## Steps to Test

Observe the results of the `--help` display for impacted commands:

- `vip dev-env`
- `vip dev-env create`
- `vip dev-env destroy`
- `vip dev-env exec`
- `vip dev-env import media`
- `vip dev-env import sql`
- `vip dev-env sql`
- `vip dev-env info`
- `vip dev-env list`
- `vip dev-env logs`
- `vip dev-env purge`
- `vip dev-env shell`
- `vip dev-env start`
- `vip dev-env stop`
- `vip dev-env sync sql`
- `vip dev-env sync`
- `vip dev-env update`
